script/seed.py

```diff
@@ -0,0 +1,554 @@
+from backend import crud
+from backend.schemas import CardCreate
+from backend.deps import get_db
+
+db = next(get_db())
+
+
+def add_card(display_name, name, type, level, definition, definition_source):
+    return crud.create_card(
+        db=db,
+        card=CardCreate(
+            display_name=display_name,
+            name=name,
+            type=type,
+            level=level,
+            definition=definition,
+            definition_source=definition_source
+        )
+    )
+
+
+def add_feeling(display_name, name, level, definition, definition_source='about:blank'):
+    return add_card(display_name, name, 'feeling', level, definition, definition_source)
+
+
+def add_need(display_name, name, level, definition, definition_source='about:blank'):
+    return add_card(display_name, name, 'need', level, definition, definition_source)
+
+
+# Sqlite version of `truncate table cards`
+db.execute('DELETE FROM cards')
```

Comment:
Instead of a drop we could upsert the cards by the unique `name` key. That way if we get a PR to change a definition after a deploy they'd be safe to run in prod without breaking the comment relationships.